src/components/display/fewlines/SnackBar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+
+import { IdentityTypes } from "../../../@types/Identity";
+
+export const SnackBar: React.FC<{ type: IdentityTypes }> = ({ type }) => {
+  let method;
+  type.toUpperCase() === IdentityTypes.EMAIL
+    ? (method = "mail")
+    : (method = "SMS");
+  return <Snack>Confirmation {method} has been sent</Snack>;
+};
+
+export default SnackBar;
+
+const Snack = styled.div`
+  visibility: visible;
+  min-width: 250px;
+  margin-left: -125px;
+  background-color: #333;
+  color: #fff;
+  text-align: center;
+  border-radius: 2px;
+  padding: 16px;
+  position: fixed;
+  z-index: 1;
+  left: 50%;
+  bottom: 30px;
```

Comment:
You need to use values from the theme. If you can't find values for you needs, feel free to add them. Just don't add random theme keys.

Reply:
OK fixed! cd6f1c4

Reply:
Here is the one that still need to be used with theme values:

```suggestion
  visibility: visible;
  min-width: 250px;
  margin-left: Maybe this? Not sure.;
  background-color: ${({ theme }) => theme.colors.black};
  color: ${({ theme }) => theme.colors.background};
  text-align: center;
  border-radius: this;
  padding: this;
  position: fixed;
  z-index: 1;
  left: 50%;
  bottom: 30px;
`;
```

Reply:
I can't see `margin left` in theme options...

Reply:
`margin` and `padding` are done with `theme.spaces`.

Reply:
Ok let's ignore that for now
https://github.com/fewlinesco/connect-account/pull/26/files#r483563388

Reply:
OK I ignore this for now